setup.py

```diff
@@ -65,103 +77,91 @@ def run(self):
             sys.exit(info)
 
 
+class PostDevelopCommand(develop):
+    """
+    Post-installation for development mode.
+    Execute manually with python setup.py develop or automatically included with
+    `pip install -e . -r dev-requirements.txt`.
+    """
+    def run(self):
+
+        # super
+        develop.run(self)
+
+        # development setup scripts (pre-requirements)
+        subprocess.call(f"scripts/installation/install_solc.py")
+        subprocess.call(".circleci/install_circle_cli.sh")
```

Comment:
Does this mean `circle_cli` is installed for everyone that does a nucypher dev installation?

Reply:
Yes - this way the pre-commit hook for circleci will not execute.

On the fence about inclusion - what do you think?

Reply:
Personally I never needed CircleCI CLI. What is a typical scenario for it?

Reply:
In this case - we're using a pre-commit hook that validates the circleci config.